index.html

```diff
@@ -1,10 +1,58 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="mdl-js">
```

Comment:
Html tag doesn't contain class attribute. See http://htmlbook.ru/html/html and remove it.